contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--3.2.0--3.3.0.sql

```diff
@@ -0,0 +1,9 @@
+-- complain if script is sourced in psql, rather than via ALTER EXTENSION
+\echo Use "ALTER EXTENSION ""babelfishpg_common"" UPDATE TO '3.3.0'" to load this file. \quit
+
+SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false);
+
+ALTER DOMAIN sys.TIMESTAMP SET NOT NULL;
```

Comment:
Please make sure to test it with both minor and major upgrade. Specifically, if a view is using this domain, what happens as part of minor version upgrade.